### PR TITLE
Add Awesome IOCs

### PR DESCRIPTION
**Pull request by sroberts**
> An awesome collection of indicators of compromise (and a few IOC related tools).

**https://github.com/sroberts/awesome-iocs**

**This is a list of places to get indicators of compromise. These are useful in helping network defenders identify known intrusions.**

### By submitting this pull request I confirm I've read and complied with the below requirements 🖖

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

## Requirements for your pull request

- I have read and understood the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- This pull request has a descriptive title.<br>For example, `Add Name of List`, not `Update readme.md` or `Add awesome list`.
- The entry in the Awesome list should:
	- Include a short description about the project/theme of the list. **It should not describe the list itself.**<br>Example: `- [Fish](…) - User-friendly shell.`, not `- [Fish](…) - Resources for Fish.`.
	- Be added at the bottom of the appropriate category.
- The list I'm submitting complies with these requirements:

## Requirements for your Awesome list

- **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
- It's the result of hard work and the best I could possibly produce.
- Non-generated Markdown file in a GitHub repo.
- **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)
- The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.
- Not a duplicate.
- Only has awesome items. Awesome lists are curations of the best, not everything.
- Includes a project logo/illustration whenever possible.
	- Either fullwidth or placed at the top-right of the readme. [(Example)](https://github.com/sindresorhus/awesome-electron)
	- The image should link to the project website or any relevant website.
	- The image should be high-DPI. Set it to maximum half the width of the original image.
- Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
- Includes the [Awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge).
	- Should be placed on the right side of the readme heading.
	- Should link back to this list.
- Has a Table of Contents section.
	- Should be named `Contents`, not `Table of Contents`.
	- Should be the first section in the list.
	- Should only have one level of sub-lists, preferably none.
- Has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license).
	- That means something like CC0, **not a code licence like MIT, BSD, Apache, etc.**
	- [WTFPL](http://www.wtfpl.net) and [Unlicense](http://unlicense.org) are not acceptable licenses.
	- If you use a license badge, it should be SVG, not PNG.
- Has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines).
	- The file should be named `contributing.md`. Casing is up to you.
- Has consistent formatting and proper spelling/grammar.
	- The link and description are separated by a dash. <br>Example: `- [AVA](…) - JavaScript test runner.`
	- The description starts with an uppercase character and ends with a period.
	- Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.
- Doesn't include a Travis badge.<br>You can still use Travis for list linting, but the badge has no value in the readme.

Go to the top and read it again.

### CODE REVIEW

**Comment by sroberts** [comment]
Is there anything I can do or add to help :ship: this?

**Comment by sroberts** [comment]
@legodude17 Is this a requirement? It's applied very inconsistently, even in this repository.

**Comment by legodude17** [comment]
> Entries have a description, unless the title is descriptive enough by itself. It rarely is though.

IMO, none of the titles are descriptive enough.

**Comment by sroberts** [comment]
@legodude17 Fair enough. I'll see what I can do, thanks for the feedback!

**Comment by sroberts** [comment]
Updated. Added descriptive titles as requested.

**Comment by sroberts** [comment]
@legodude17 Hoping this is ready to go now!

**Comment by sindresorhus** [comment]
Hey, sorry for the lack of response, please see #1363.

**Comment by desireevl** [comment]
Some suggested edits to satisfy requirement guidelines:
- If you use a license badge, it should be SVG, not PNG.
- The description starts with an uppercase character and ends with a period.

**Comment by uraimo** [comment]
Other than what Desiree said, please note that the awesome logo should be on the right of the title and that the rules now require that no travis/circleci badge should be present.
I'd move the CC license badge (in svg) to the bottom under a "License" heading.
After these few things you are ready to go, imho.

**Comment by sroberts** [comment]
@desireevl There is no SVG badge for License Zero.  PNG it is. I did fix the period on the description.

**Comment by sroberts** [comment]
@uraimo Moved the license info to a separate section at the bottom and removed the CI badge.

**Comment by sroberts** [comment]
I'm pretty sure I should be good at this point.

**Comment by sindresorhus** [comment]
```
❯ awesome-lint https://github.com/sroberts/awesome-iocs
✖ Linting

  README.md:1:17
  ✖   1:17   Invalid badge source                                                            remark-lint:awesome/badge
  ✖   7:1    ToC item "IOCs" link "https://github.com/sroberts/awesome-iocs#iocs" not found  remark-lint:awesome/toc
  ✖  19:58   List item description must end with proper punctuation                          remark-lint:awesome/list-item
  ✖  20:70   List item description must end with proper punctuation                          remark-lint:awesome/list-item
  ✖  21:52   List item description must end with proper punctuation                          remark-lint:awesome/list-item
  ✖  22:76   List item description must end with proper punctuation                          remark-lint:awesome/list-item
  ✖  23:64   List item description must end with proper punctuation                          remark-lint:awesome/list-item
  ✖  24:84   List item description must end with proper punctuation                          remark-lint:awesome/list-item
  ✖  26:58   List item description must end with proper punctuation                          remark-lint:awesome/list-item
  ✖  27:50   List item description must end with proper punctuation                          remark-lint:awesome/list-item
  ✖  28:60   List item description must end with proper punctuation                          remark-lint:awesome/list-item
  ✖  29:82   List item description must end with proper punctuation                          remark-lint:awesome/list-item
  ✖  30:56   List item description must end with proper punctuation                          remark-lint:awesome/list-item
  ✖  34:53   List item description must end with proper punctuation                          remark-lint:awesome/list-item
  ✖  35:64   List item description must end with proper punctuation                          remark-lint:awesome/list-item
  ✖  39:62   List item description must end with proper punctuation                          remark-lint:awesome/list-item
  ✖  40:84   List item description must end with proper punctuation                          remark-lint:awesome/list-item
  ✖  41:70   List item description must end with proper punctuation                          remark-lint:awesome/list-item
  ✖  51:50   List item description must end with proper punctuation                          remark-lint:awesome/list-item
  ✖  53:54   List item description must end with proper punctuation                          remark-lint:awesome/list-item
  ✖  59:104  List item description must end with proper punctuation                          remark-lint:awesome/list-item
  ✖  62:91   List item description must end with proper punctuation                          remark-lint:awesome/list-item
  ✖  63:45   List item description must end with proper punctuation                          remark-lint:awesome/list-item
  ✖  67:2    License image must be SVG                                                       remark-lint:awesome/license
```

https://github.com/sindresorhus/awesome-lint

**Comment by sindresorhus** [comment]
> Includes a succinct description of the project/theme at the top of the readme. (Example) - https://github.com/sindresorhus/awesome/blob/master/pull_request_template.md#requirements-for-your-awesome-list

**Comment by sindresorhus** [comment]
You're writing it incorrectly, it's `IoC`, not `IOC`. And why the `s` at the end?

File: readme.md [hunk 1]
```diff
@@ -430,6 +430,7 @@
 - [Vehicle Security and Car Hacking](https://github.com/jaredthecoder/awesome-vehicle-security)
 - [Web Security](https://github.com/qazbnm456/awesome-web-security) - Security of web apps & services.
 - [Lockpicking](https://github.com/meitar/awesome-lockpicking) - The art of unlocking a lock by manipulating its components without the key.
+- [IOCS](https://github.com/sroberts/awesome-iocs) - Indicators of Compromise for detecting intrusions.
```
Comment (sindresorhus) [inline readme.md]: I think the title should be `Indicator of compromise (IoC)`. `IoC` is too ambiguous and meaningless just alone. It also needs a better description.

**Comment by sindresorhus** [comment]
You also need to fix the merge conflict.

**Comment by sroberts** [comment]
@sindresorhus First of all thank you so much for the actionable feedback (as well as running an amazing project). I'll get these things fixed.

**Comment by sindresorhus** [comment]
The merge conflict is not fixed correctly. Look at the diff.

**Comment by sindresorhus** [comment]
https://github.com/sindresorhus/awesome/pull/1305#issuecomment-466011305 is not fixed.

**Pull request closed without merging**